frontend_app_file/test/tlmMock.js

```diff
@@ -0,0 +1,8 @@
+export const registerLiveMessageHandlerListMock = (callback) => (tlmList) => {
```

Comment:
rename file tracimLiveMessageMock.js please, avoid abbreviation in file names.
as a general rule, avoid abbreviations